grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -2814,4 +2815,23 @@ class StockMovementService {
         }
         return true
     }
+
+    static String getDisplayStatus(Shipment shipment, Requisition requisition) {
```

Comment:
@alannadolny Actually now when I see this method, I think it should be directly in the StockMovement instead of here (if you only use it in the `StockMovement` - initially I thought it might end up being used in the `OutboundStockMovement` too), and you won't have to pass shipment and requisition in params